app/controllers/podcast_metrics_controller.rb

```diff
@@ -18,13 +20,11 @@ def downloads
           .load_async
     end
 
-    @downloads = {
-      rollups: @downloads_within_date_range,
-      color: primary_blue
-    }
+    @downloads = single_rollups(@downloads_within_date_range)
 
-    render partial: "downloads_card", locals: {
-      podcast: @podcast,
+    render partial: "metrics/downloads_card", locals: {
+      url: downloads_podcast_metrics_path(podcast: @podcast, date_start: @date_start, date_end: @date_end, interval: @interval),
+      form_id: "podcast_downloads_metrics",
```

Comment:
`url` and `form_id` local variables to specify routing on the now generalized `metrics/downloads_card`.